src/app/shared/components/select/select.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Output, EventEmitter, Component, Input, forwardRef } from '@angular/core';
```

Comment:
Do we can put this control into form-input, I see it the same

Reply:
the select component have some different about input with the option array. so I think separate to new component for easier to  maintain thanks @thienkieu

Reply:
we can add more options for form-input, when type is 'select box' we can renter it